test/Microsoft.Extensions.Internal.Test/Microsoft.Extensions.Internal.Test.csproj

```diff
@@ -5,6 +5,7 @@
   <PropertyGroup>
     <TargetFrameworks>netcoreapp2.0;net461</TargetFrameworks>
     <TargetFrameworks Condition=" '$(OS)' != 'Windows_NT' ">netcoreapp2.0</TargetFrameworks>
+    <DebugType>portable</DebugType>
```

Comment:
The SDK generates a full pdb for cross-compiled applications (on Windows?). CoreCLR can't seem to read a full pdb and our pdb reader only works one way and we intend to keep it so. Forcing it to portable since that's what most users would have.